contracts/contracts/GrantRoundManager.sol

```diff
@@ -74,17 +75,20 @@ contract GrantRoundManager {
   function createGrantRound(
     address _owner,
     address _payoutAdmin,
+    IERC20 _matchingToken,
     GrantRegistry _registry,
     uint256 _startTime,
     uint256 _endTime,
     string memory _metaPtr,
     uint256 _minContribution
   ) external {
+    require(_matchingToken.totalSupply() > 0, "GrantRoundManager: Invalid matching token");
```

Comment:
Adding similar check to the donation token supply check in this contracts constructor